tests/behaviour/util.rs

```diff
@@ -43,14 +43,14 @@ pub fn iter_table(step: &Step) -> impl Iterator<Item = &str> {
     step.table().unwrap().rows.iter().flatten().map(String::as_str)
 }
 
-pub fn iter_table_map(step: &Step) -> impl Iterator<Item = HashMap<&String, &String>> {
+pub fn iter_table_map(step: &Step) -> impl Iterator<Item = HashMap<&str, &str>> {
     let (keys, rows) = step.table().unwrap().rows.split_first().unwrap();
-    rows.iter().map(|row| keys.iter().zip(row).collect())
+    rows.iter().map(|row| keys.iter().map(String::as_str).zip(row.into_iter().map(String::as_str)).collect())
```

Comment:
A little ugly, but that's something we can deal with. I'd push the conversion into a single map at the end, which'd be closer to what you had before:
```rust
rows.iter().map(|row| keys.iter().zip(row).map(|(k, v)| (k.as_str(), v.as_str())).collect())
```